tests/test_conditionals.py

```diff
@@ -160,5 +163,48 @@ def test_whiten(self):
             assert_allclose(var_difference, 0, atol=4)
 
 
+@pytest.mark.parametrize("full_cov", [True, False])
+@pytest.mark.parametrize("features_inducing_points", [False, True])
+def test_base_conditional_vs_ref(session_tf, full_cov, features_inducing_points):
+    """
+    Test that conditionals agree with a slow-but-clear numpy implementation
```

Comment:
I think this is great, would be even better to have the slow-but-clear numpy implementation in its own function (just to make the test a bit easier to follow and to decouple setup from calling conditionals).

Reply:
yes, you are right! These tests can use a bit of restructuring